bindings/c/test/apitester/TesterBlobGranuleErrorsWorkload.cpp

```diff
@@ -152,7 +152,7 @@ class BlobGranuleErrorsWorkload : public ApiWorkload {
 		auto success = std::make_shared<bool>(false);
 		execOperation(
 		    [begin, end, blobbify, success](auto ctx) {
-			    fdb::Future f = blobbify ? ctx->db().blobbifyRange(begin, end).eraseType()
+			    fdb::Future f = blobbify ? ctx->db().blobbifyRange(begin, end, false).eraseType()
```

Comment:
Here also probably both true and false case needs to be tested, so that we are sure that our futures do not get stuck in error scenarios